development/src/logic/pieces.ts

```diff
@@ -78,8 +78,36 @@ export type Square = {
 };
 
 export class Pawn extends Piece {
+
+  enPassant: boolean;
+
   constructor(position: Position, player: Player) {
     super(position, player, pawnResource, 'Pawn');
+    this.enPassant = false;
+  }
+
+  enPassantCheck(targetPosition: Position, draggedPiece: Piece){
+
```

Comment:
the function should have a specific return type, in this case I believe it would be `enPassantCheck(targetPosition: Position, draggedPiece: Piece): boolean {`